portal-api/src/v1/controllers/utilisation-report-service/fee-record-corrections/get-fee-record-correction.controller/index.test.ts

```diff
@@ -0,0 +1,129 @@
+import httpMocks, { MockResponse } from 'node-mocks-http';
+import { AxiosResponse, HttpStatusCode, AxiosError } from 'axios';
+import { Response } from 'express';
+import { getFeeRecordCorrection, GetFeeRecordCorrectionRequest } from '.';
+import api from '../../../../api';
+import { aGetFeeRecordCorrectionResponseBody } from '../../../../../../test-helpers/test-data/get-fee-record-correction-response-body';
+
+jest.mock('../../../../api');
+
+console.error = jest.fn();
+
+describe('get-fee-record-correction.controller', () => {
+  describe('getFeeRecordCorrection', () => {
+    const bankId = '123';
+    const correctionId = 7;
+
+    const aValidRequestQuery = () => ({ correctionId: correctionId.toString(), bankId });
+
+    let req: GetFeeRecordCorrectionRequest;
+    let res: MockResponse<Response>;
+
+    beforeEach(() => {
+      req = httpMocks.createRequest<GetFeeRecordCorrectionRequest>({
+        params: aValidRequestQuery(),
+      });
+      res = httpMocks.createResponse();
+
+      jest.mocked(api.getFeeRecordCorrectionById).mockResolvedValue(aGetFeeRecordCorrectionResponseBody());
+    });
+
+    afterEach(() => {
+      jest.resetAllMocks();
+    });
+
+    it(`should return a ${HttpStatusCode.Ok} status code if the api request is successful`, async () => {
+      // Act
+      await getFeeRecordCorrection(req, res);
+
+      // Assert
+      expect(res._getStatusCode()).toEqual(HttpStatusCode.Ok);
```

Comment:
I *think* this test is only passing because you happen to use the same bank id for the user as `aGetFeeRecordCorrectionResponseBody` uses, but we don't want our tests to rely on values in `a` methods since they just give us 'a' object of that type, so please can we explicitly make them match in our test set up?